--- a/Makefile
+++ b/Makefile
@@ -71,7 +71,7 @@ else ifeq ($(SCENIC_LOCAL_TARGET),bcm)
 else ifeq ($(SCENIC_LOCAL_TARGET),drm)
 	# drm is the forward looking default
-	LDFLAGS += -lGLESv2 -lEGL -lm -lvchostif -ldrm -lgbm
+	LDFLAGS += -lGLESv2 -lEGL -lm -ldrm -lgbm
 	CFLAGS ?= -O2 -Wall -Wextra -Wno-unused-parameter -pedantic
 	CFLAGS += -std=gnu99
--- a/c_src/image.c
+++ b/c_src/image.c
@@ -22,7 +22,7 @@ limitations under the License.
 #define HASH_ID(id) tommy_hash_u32( 0, id.p_data, id.size )


-#define REPEAT_XY   (NVG_IMAGE_REPEATX | NVG_IMAGE_REPEATY)
+#define REPEAT_XY   0


 tommy_hashlin   images = {0};
